protostar/testing/cairo_test_runner.py

```diff
@@ -101,6 +102,14 @@ def worker(cls, args: "TestRunner.WorkerArgs"):
             )
         )
 
+    async def _build_execution_state(self, test_config: TestConfig):
+        state = await TestExecutionState.for_cairo_execution(
+            test_config=test_config,
+            project_compiler=self.project_compiler,
+        )
+        # TODO: Execute __setup__ using test_execution_state & cairo_setup_execution_environment
```

Comment:
We have already two TODOs about `__setup__` in this file.

Reply:
That's not true, one is about `setup_case` and second is about `__setup__`, both in respective places they should be plugged into

Reply:
Oh ok I see, my bad